integration-tests/tsconfig.json

```diff
@@ -1,74 +1,71 @@
 {
-  "compilerOptions": {
```

Comment:
Prettier is not making this file nicer. Let's leave out `tsconfig.json` files.

Reply:
Excluded the `tsconfig.json` and the WebPack one